src/dotnet-roslyn-tools/CreateReleaseTags/SdkReleaseTagger.cs

```diff
@@ -112,6 +112,38 @@ static string ConvertVersionToBuildId(string version)
         }
     }
 
+    public override async Task<SdkBuildInformation?> TryGetBuildAsync(RemoteConnections connections, IProduct product, SdkBuildInformation vmrBuild)
+    {
+        // This is used when the SHA returned from the Release build does not exist in the target repo. In SDK scenarios it
+        // gives us a chance to query the dotnet VMR and identify the SHA for our repo.
+        VmrSourceManifest? sourceManifest = null;
+
+        try
+        {
+            sourceManifest = await connections.GitHubClient
+                .GetFromJsonAsync<VmrSourceManifest?>($"https://raw.githubusercontent.com/dotnet/dotnet/{vmrBuild.CommitSha}/src/source-manifest.json");
+        }
+        catch (Exception) { }
```

Comment:
Should this log anything?  For example if we get rate limited it seems like we'd never see it.